edx_event_bus_kafka/internal/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Most of the implementation of the package is here. See this ADR in a different project for the reasoning:
+
+https://github.com/openedx/edx-django-utils/blob/master/docs/decisions/0004-public-api-and-app-organization.rst
+
+Public API will be in the `edx_event_bus_kafka` module for the most part.
```

Comment:
Maybe something like:
```suggestion
Most of the implementation of the package is here and is internal-only.

Public API will be in the `edx_event_bus_kafka` module for the most part.

See this ADR for the reasoning:
https://github.com/openedx/event-bus-kafka/blob/master/docs/decisions/0006-public-api-and-app-organization.rst
```